addons/mail/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js

```diff
@@ -459,8 +459,8 @@ QUnit.test('chat window: basic rendering', async function (assert) {
     );
     assert.strictEqual(
         chatWindowHeader.querySelectorAll(`:scope .o_ChatWindowHeader_command`).length,
-        5,
-        "should have 5 commands in header part"
+        6,
+        "should have 6 commands in header part"
```

Comment:
Below assertions told which specific commands were in the "5" asserted here.
Now that it becomes 6, should add assertion with the 6th newly introduced one.